flang/test/Lower/OpenMP/Todo/omp-default-clause-inner-loop.f90

```diff
@@ -1,7 +1,12 @@
 ! This test checks the lowering of parallel do
 
-! RUN: %flang_fc1 -emit-fir -flang-deprecated-no-hlfir -fopenmp %s -o - | FileCheck %s
-! RUN: bbc -fopenmp -emit-fir -hlfir=false %s -o - | FileCheck %s
+! RUN: %flang_fc1 -emit-fir -flang-deprecated-no-hlfir -fopenmp \
```

Comment:
I did not fix the expecataions since we have quite a few tests that have delayed privatization enabled. Let me know if you prefer to switch it on in the following tests as well and instead update the checks to verify the outlined privatizers.

Reply:
For tests for which similar tests exist elsewhere it is not important. But for others it is always good to test the new flow. It is less important to test the contents of the `privateclause` operation, but would be good to check that the private operand is created and the block argument is used inside the parallel region.